modules/images-imagestream-import-import-mode.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+// * openshift_images/image-streams-managing.adoc
+
+:_content-type: PROCEDURE
+[id="images-imagestream-import-import-mode_{context}"]
+= Working with manifest lists
+
+You can import all manifests of a manifest list when using `oc import-image` or `oc tag` CLI commands by adding the `--import-mode` flag.
+
+.Procedure
+
+* Entering the following command creates an image stream that includes multi-architecture images, and sets the import mode to `PreserveOriginal`. This ensures that the manifest list and all of its sub-manifests are imported. For example:
++
+[source,terminal]
+----
+$ oc import-image <multiarch-image-stream-tag>  --from=<registry>/<username>/<image-name> \
+--import-mode='PreserveOriginal' --reference-policy=local --confirm
+----
```

Comment:
Do we need add a example command line here?

Reply:
And example output

Reply:
<username> should replace with <project_name>